### ISSUE TITLE
TinyMCE Usage Varies Between EDIT and ADD Actions

### ISSUE THREAD

**Issue by maxdumas** [TechAtNYU/intranet#119]
To reproduce, see [edit](http://intranet.sexy/#/r/related-clubs/edit/548112cac0e3d27ee9073d26) for related versus [add](http://intranet.sexy/#/r/related-clubs/add).

The "Name" field on the EDIT page uses a TinyMCE input (which is incorrect), while the ADD page uses a textarea, which is correct. This must have something to do with some incorrect binding of data when existing values are being loaded for the edit page, or some other incongruity between the two.

**Comment by ethanresnick** [comment]
On my machine at least, the behavior on the edit and add pages is the same: both show the WYSIWG when they're not supposed to, _but only about half the time_. Try refreshing each page a bunch of times and I think you'll see that you can sometimes get the WYSIWG to show up (or not) on each. The data binding issue appears to be occurring now only when the WYSIWG is showing incorrectly, but I'm not 100% sure about that.

**Comment by maxdumas** [comment]
What the *shit*. My only explanation for this is that the server running the API or serving the intranet has developed free will and so is no longer deterministic.

**Comment by ethanresnick** [comment]
Yeah. It's fucking weird. You can reproduce it, though, right?

**Comment by maxdumas** [comment]
Yeah I can. You're right, it's about 50% of the time. I'm going to try to look into this tomorrow.

**Comment by ethanresnick** [comment]
:+1:

**Comment by freialobo** [comment]
"My only explanation for this is that the server running the API or serving the intranet has developed free will and so is no longer deterministic." is the best thing ever omg.

**Comment by jhishan** [comment]
^LOL, we have some AI stuff going on here

**Comment by maxdumas** [comment]
Before I forget since I haven't had time to tackle this: The issue with this is that for some reason tinyMCE is converting all textarea fields on the page into tinyMCE inputs automatically, which I think screws up our model binding majorly and the API doesn't like it either. I couldn't figure out why this is happening — it doesn't look like we're setting anything on tinyMCE that would cause this. But this is definitely what is wrong.
